microreactors/mrad/steady/hpc_tests

```diff
@@ -0,0 +1,17 @@
+[Tests]
+  [coupled_simulation]
+    type = CSVDiff
+    input = 'HPMR_dfem_griffin_ss.i'
+    csvdiff = 'mrad_coupled_out.csv'
+    executable_pattern = 'dire_wolf*'
+    cli_args = "Executioner/richardson_max_its=2 Executioner/richardson_abs_tol=1e-1 Outputs/csv=true "
+               "Outputs/file_base=mrad_coupled_out Outputs/exodus=false MultiApps/bison/cli_args='Outputs/exodus=false'"
+    # To avoid running out of memory
+    mesh_mode = distributed
```

Comment:
Just pass `--distributed-mesh` in cli args to force this. otherwise it'll be a skip

Reply:
You still need to add --distributed-mesh to cli_args

Reply:
I dont recall if I was running out of memory. but yeah next step if I am.

Reply:
It did

```
microreactors/mrad/steady.coupled_simulation: Job 1535050 has state "OUT_OF_MEMORY"
microreactors/mrad/steady.coupled_simulation: ################################################################################
microreactors/mrad/steady.coupled_simulation: 
microreactors/mrad/steady.coupled_simulation: ################################################################################
microreactors/mrad/steady.coupled_simulation: End job output
microreactors/mrad/steady.coupled_simulation: 
[278.0s]    ERROR microreactors/mrad/steady.coupled_simulation FAILED (SLURM ERROR: OUT_OF_MEMORY) [min_cpus=48,job=1535050]
```